### PR TITLE
dotCMS/core#15513 Add toast notification handler to angular

### PR DESCRIPTION
**Pull request by freddyucv**

### CODE REVIEW

**Review by jgambarios — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/api/system/event/message/MessageSeverity.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.dotcms.api.system.event.message;
+
+import com.dotcms.api.system.event.message.builder.SystemMessage;
+
+/**
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/api/system/event/message/MessageSeverity.java]: Add some doc

File: dotCMS/src/main/java/com/dotcms/api/system/event/message/MessageType.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.dotcms.api.system.event.message;
+
+import com.dotcms.api.system.event.message.builder.SystemMessage;
+
+/**
+ * {@link SystemMessage} type
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/api/system/event/message/MessageType.java]: Add doc

File: dotCMS/src/main/java/com/dotcms/api/system/event/message/SystemMessageEventUtil.java [hunk 3]
```diff
@@ -139,7 +148,7 @@ public void pushRichMediaEvent (final Object richMediaMessage, final String user
      */
     public void pushRichMediaEvent (final Object richMediaMessage, final List<String> users, final String... portletIds) {
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/api/system/event/message/SystemMessageEventUtil.java]: delete this

File: dotCMS/src/main/java/com/dotcms/api/system/event/message/SystemMessageEventUtil.java [hunk 4]
```diff
@@ -101,8 +105,13 @@ public void pushSimpleErrorEvent (final ErrorEntity error, final String userId,
      *                   otherwise even if the message is for that particular user, if it is not in the portlet will be skipped. (note: this validation will happens on the client, Angular or whatever is the consumer)
      */
     public void pushSimpleErrorEvent (final ErrorEntity error, final List<String> users, final String... portletIds) {
+        final SystemMessage systemMessage = SystemMessageBuilder.getInstance()
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/api/system/event/message/SystemMessageEventUtil.java]: I think this should be a new instance

File: dotCMS/src/main/java/com/dotcms/api/system/event/message/builder/SystemMessageBuilder.java [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+public class SystemMessageBuilder {
+    private Object message;
+    private String[] portletIdList;
+    private long life = 3000;
+    private MessageSeverity severity = MessageSeverity.INFO;
+    private MessageType type = MessageType.SIMPLE_MESSAGE;
+
+    private SystemMessageBuilder(){}
+
+    private static class SingletonHolder {
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/api/system/event/message/builder/SystemMessageBuilder.java]: this should be not a singleton

**Review by jdotcms — APPROVED** [review]

**Pull request merged**